src/sort-transactions.js

```diff
@@ -19,8 +19,8 @@ module.exports = function sortTransactions(arr) {
     const methodIndexA = sortedMethods.indexOf(a.request.method);
     const methodIndexB = sortedMethods.indexOf(b.request.method);
 
-    if (!(methodIndexA < methodIndexB)) { return -1; }
-    if (!(methodIndexA > methodIndexB)) { return 1; }
+    if (methodIndexA < methodIndexB) { return -1; }
+    if (methodIndexA > methodIndexB) { return 1; }
```

Comment:
This change looks suspicious. Removal of the `!` flips the meaning of both the lines. Is that intentional?

Reply:
Well, since the original (decaffed) code looked like this:

```javascript
switch (false) {
  case !(methodIndexA < methodIndexB): return -1;
  case !(methodIndexA > methodIndexB): return 1;
  case methodIndexA !== methodIndexB:
    return a['_index'] - b['_index'];
}
```
...it is correct. Also, test were failing for previous version.